packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3158,7 +3162,12 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
 
     final bool wasSemanticsBoundary = _semantics != null && (_cachedSemanticsConfiguration?.isSemanticBoundary ?? false);
     _cachedSemanticsConfiguration = null;
-    bool isEffectiveSemanticsBoundary = _semanticsConfiguration.isSemanticBoundary && wasSemanticsBoundary;
+    // Merger may produce sibling node to be attached to the parent of this
```

Comment:
nit: eliminate the word "Merger" here since that's not what this is called anymore?

Reply:
ah sorry for this, my IDE lied to me again, I did a search all on keyword merger, but it still not catching this and several instances. Will fix